### DISCUSSION TITLE
一直显示IP没变化

### DISCUSSION THREAD

**Discussion by 2451760252** [jeessy2/ddns-go#467] [Q&A]
一直显示IP没变化但是DNS服务商，软件，实际IP三个IP不一样
重装过几次但是过一会又不正常了
![image](https://user-images.githubusercontent.com/120788853/208244116-2e7436a0-9ae4-447c-9e2e-9476fff4e829.png)
![image](https://user-images.githubusercontent.com/120788853/208243845-0415620b-7be8-4a09-8e2f-d516abf536a5.png)
![image](https://user-images.githubusercontent.com/120788853/208243864-a22a829d-19b7-464f-ada8-1984f1672be7.png)

**Comment by jeessy2** [comment]
IPv6地址有多个，看上去是不一样

**Reply by 2451760252** [reply]
但是[DDNS-GO]和DNS显示的IP无法访问
![image](https://user-images.githubusercontent.com/120788853/208245690-c7eccbdf-7ed2-4a77-ad5d-3f97e847d849.png)

**Reply by jeessy2** [reply]
只要简析上去了，就和ddns无关了。检查你防火墙

**Reply by 2451760252** [reply]
没解析上去

**Reply by jeessy2** [reply]
看日志解析上去了。你登录你服务商看下

**Reply by 2451760252** [reply]
解析上了但是IP不对

**Reply by 2451760252** [reply]
它的IP一直是2408:8215:5511:f150:fa68:19f0:5507:5e89但是真实IP是2408:8215:5511:f150:f47d:97e2:f937:53d3

**Reply by jeessy2** [reply]
IPv6地址有几个，都可以用。要指定从网卡获取

**Reply by 1123123w** [reply]
> IPv6地址有几个，都可以用。要指定从网卡获取

歪个楼，不知道TTL时间能否自定义，或者设置更长？目前最长是1小时。

**Reply by WaterLemons2k** [reply]
> 歪个楼，不知道TTL时间能否自定义，或者设置更长？目前最长是1小时。

TTL 应该在配置文件 `.ddns_go_config.yaml` 里就可以修改了，像这样：

```YAML
ttl: ""
```

**Reply by WaterLemons2k** [reply]
简陋的修改 TTL 值的 JS：
```javascript
document.getElementById('ttl').options[document.getElementById('ttl').selectedIndex].value = prompt()
```

**可能会出现意想不到的效果！**

**Reply by 1123123w** [reply]
JS在哪里修改呢，刚刚看了看直接修改配置文件的时长，貌似不生效

**Reply by WaterLemons2k** [reply]
> JS在哪里修改呢，刚刚看了看直接修改配置文件的时长，貌似不生效

控制台里吧。另外如果不改成现有的值会显示空白：
https://github.com/jeessy2/ddns-go/blob/d4bb61e3467d2aa69be064a1a6dceafef760ce1d/web/writing.html#L152-L159

最好改成域名服务商现有的 TTL。
